api/server/v1/marshaler.go

```diff
@@ -55,7 +54,7 @@ type CustomMarshaler struct {
 
 func (c *CustomMarshaler) NewDecoder(r io.Reader) runtime.Decoder {
 	return CustomDecoder{
-		jsonDecoder: json.NewDecoder(r),
+		jsonDecoder: jsoniter.NewDecoder(r),
```

Comment:
NewDecoder and RawMessage are aliased in our JSON package, we can also move `Decoder` inside it and then refer to our package here?